### PR TITLE
Second round audit fixes

### PR DESCRIPTION
**Pull request by kovalgek**
- added permission for pushing token rate from TokenRateNotifier
- added additional check for time when resuming

### CODE REVIEW

File: contracts/lido/TokenRateNotifier.sol [hunk 1]
```diff
@@ -94,6 +102,9 @@ contract TokenRateNotifier is Ownable, IPostTokenRebaseReceiver {
         uint256, /* postTotalEther     */
         uint256  /* sharesMintedAsFees */
     ) external {
+        if (msg.sender != AUTHORIZED_REBASE_CALLER) {
```
Comment (TheDZhon) [inline contracts/lido/TokenRateNotifier.sol]:
```suggestion
        if (msg.sender != LIDO) {
```

Comment (TheDZhon) [reply contracts/lido/TokenRateNotifier.sol]: may also add a comment `see Lido._completeTokenRebase`

**Review by TheDZhon — APPROVED** [review]
LGTM except only for a few suggestions :pray:

**Review by arwer13 — APPROVED** [review]
LGTM

**Pull request merged**